hazelcast/src/main/java/com/hazelcast/map/impl/operation/MapOperation.java

```diff
@@ -420,7 +420,8 @@ private void publishWanUpdateInternal(Data dataKey, Object value, boolean hasLoa
         ExpiryMetadata expiryMetadata = recordStore.getExpirySystem().getExpiryMetadata(dataKey);
         WanMapEntryView<Object, Object> entryView = createWanEntryView(
                 toHeapData(dataKey), dataValue, record, expiryMetadata,
-                getNodeEngine().getSerializationService());
+                getNodeEngine().getSerializationService(),
+                mapContainer.getMapConfig().isPerEntryStatsEnabled());
```

Comment:
possibly no need to change method signature. Please see this: https://github.com/hazelcast/hazelcast/pull/23279/files#r1059394042

Reply:
https://github.com/hazelcast/hazelcast/pull/23279/commits/fc2720695ff701179f9c306ae395d23265ecfd68